nv2a-trace.py

```diff
@@ -49,8 +43,26 @@ def __init__(self):
 
 xbox_helper = XboxHelper(xbox)
 
-def main():
+def main(args):
+
+  # Create output folder
+  try:
+    os.mkdir(args.out)
+    Trace.OutputDir = args.out
```

Comment:
mkdir throws FileExistsError then OutputDir is never set. Intentional?

If you want to tolerate existing dirs then probably want to first check with  `os.path.isdir` or use `os.mkdirs(dirname, exist_ok=True)`

Reply:
Yeah, this is another one that's fixed a couple PRs into the future when the Trace.OutputDir becomes an instance variable within the Tracer.